src/display/editor/tools.js

```diff
@@ -797,6 +801,14 @@ class AnnotationEditorUIManager {
     }
   }
 
+  async guess(data) {
```

Comment:
This feels like a very "general" name, given the primary functionality in this class, how about e.g. the following?
```suggestion
  async mlGuess(data) {
```